server/src/main/java/org/jboss/as/server/ServerEnvironment.java

```diff
@@ -304,6 +304,7 @@ public ProcessType getProcessType() {
     private final boolean startGracefully;
     private final GitRepository repository;
     private final Stability stability;
+    private final Set<Stability> stabilities;
```

Comment:
This field shouldn't be needed as this.productConfig can be called to get the stabilities set.